backend/bitboxbase/handlers/handlers.go

```diff
@@ -129,25 +264,158 @@ func (handlers *Handlers) getVerificationProgressHandler(r *http.Request) (inter
 	}, nil
 }
 
-func (handlers *Handlers) postConnectElectrumHandler(r *http.Request) (interface{}, error) {
+func (handlers *Handlers) postConnectElectrumHandler(_ *http.Request) (interface{}, error) {
 	err := handlers.base.ConnectElectrum()
 	if err != nil {
 		return bbBaseError(err, handlers.log), nil
 	}
-	return map[string]interface{}{"success": true}, nil
+	return map[string]interface{}{
+		"success": true,
+	}, nil
+}
+
+func (handlers *Handlers) postResyncBitcoinHandler(_ *http.Request) (interface{}, error) {
+	handlers.log.Debug("postResyncBitcoinHandler")
+	err := handlers.base.ResyncBitcoin()
+	if err != nil {
+		return bbBaseError(err, handlers.log), nil
+	}
+	return map[string]interface{}{
+		"success": true,
+	}, nil
+}
+
+func (handlers *Handlers) postReindexBitcoinHandler(_ *http.Request) (interface{}, error) {
+	handlers.log.Debug("postReindexBitcoinHandler")
+	err := handlers.base.ReindexBitcoin()
+	if err != nil {
+		return bbBaseError(err, handlers.log), nil
+	}
+	return map[string]interface{}{
+		"success": true,
+	}, nil
+}
+
+func (handlers *Handlers) postEnableTorHandler(r *http.Request) (interface{}, error) {
+	handlers.log.Debug("Enable Tor")
+	var toggleAction string
+	if err := json.NewDecoder(r.Body).Decode(&toggleAction); err != nil {
+		return nil, errp.WithStack(err)
+	}
+	if err := handlers.base.EnableTor(rpcmessages.ToggleSetting(toggleAction)); err != nil {
+		return bbBaseError(err, handlers.log), nil
+	}
+	return map[string]interface{}{
+		"success": true,
+	}, nil
+}
+
+func (handlers *Handlers) postEnableTorMiddlewareHandler(r *http.Request) (interface{}, error) {
+	handlers.log.Debug("Enable Tor for middleware")
+	var toggleAction string
```

Comment:
Why not use JSON here as elsewhere? (examples see ctrl+f 'payload') Not sure what to pass here.

Reply:
No strong feelings about this really, but since we're just sending one value "enable"/"disable", makes sense to me to just send that as a string without additional key/value pairs. 
So you can just test it like this:
`curl -d '"enable"' -H 'Content-Type: text/json' http://localhost:8082/api/bitboxbases/localhost:8845/enable-tor`

Reply:
> Also done like this in many other places in the app

I'm fine with it then.

Reply:
Changed back to `bool` now with `type ToggleSetting bool` in rpcmessages.go, so disregard.

But to address your feedback, I wouldn't consider it technical debt in the app. We regularly use it for single value post requests and since the Content-Type is still `text/json` it can be decoded the same way